lib/setup-terraform.js

```diff
@@ -163,7 +163,7 @@ async function run () {
   try {
     // Gather GitHub Actions inputs
     const version = core.getInput('terraform_version');
-    const credentialsHostname = core.getInput('cli_config_credentials_hostname');
+    const credentialsHostname = core.getInput('cli_config_credentials_hostname') || 'app.terraform.io';
```

Comment:
This change isn't needed since GitHub Actions will set the default value for you. Similar to how the `terraform_version` and `terraform_wrapper` inputs work.